packages/Pill/src/Pill.styles.js

```diff
@@ -1,101 +1,126 @@
-import { css } from "styled-components";
+import styled from "styled-components";
 import stylers from "@paprika/stylers";
 import tokens from "@paprika/tokens";
+import RawButton from "@paprika/raw-button";
 
 const pillColorStyles = {
-  black: css`
+  black: `
```

Comment:
Probably not a big deal in these cases, you could leave it as is, but the `css` helper was there for that same syntax highlighting reason.